tmt/steps/discover/fmf.py

```diff
@@ -3,16 +3,19 @@
 import re
 import shutil
 import subprocess
+from typing import List, Optional, cast
 
 import click
 import fmf
 
 import tmt
 import tmt.beakerlib
+import tmt.options
 import tmt.steps.discover
+from tmt.base import Test
 
 
-class DiscoverFmf(tmt.steps.discover.DiscoverPlugin):
+class DiscoverFmf(tmt.steps.discover.DiscoverPlugin):  # type: ignore[misc]
```

Comment:
Also blocked on https://github.com/teemtee/tmt/pull/1333 due to this `# type: ignore`, right? It is also ready for review so we should focus on that PR first.

Reply:
That's not exactly blocking it, it can still be merged and there are a lot of places with such things that are already merged, i.e https://github.com/teemtee/tmt/blob/main/tmt/steps/provision/artemis.py#L234 .
I added a comment to remove it once its superclass is typed